lib/arrow_web/views/disruption_view/calendar.ex

```diff
@@ -51,6 +52,14 @@ defmodule ArrowWeb.DisruptionView.Calendar do
     end)
   end
 
+  defp row_status(row_approved) do
+    if row_approved do
+        "approved"
+    else
+        "pending"
+    end
+  end
```

Comment:
suggestion: might be better as a pattern match?
```suggestion
  defp row_status_class(true), do: "status-approved"
  defp row_status_class(false), do: "status-pending"
```